state/transaction.go

```diff
@@ -574,12 +573,16 @@ func (s *State) ParseTheTraceUsingTheTracer(evm *fakevm.FakeEVM, trace instrumen
 				}
 				tracer.CaptureExit(step.ReturnData, previousStepGasCost.Uint64(), previousError)
 			} else {
+				value := hex.DecodeBig(step.Contract.Value)
+				if previousStep.OpCode == "STATICCALL" {
+					value = nil
+				}
```

Comment:
just review it but i think this is true also for delegateCalls ( in the callTracer traces)

Reply:
It's not that simple, we need to investigate this further